src/hotspot/cpu/riscv/c1_MacroAssembler_riscv.cpp

```diff
@@ -324,7 +324,7 @@ void C1_MacroAssembler::verified_entry(bool breakAtEntry) {
   // must ensure that this first instruction is a J, JAL or NOP.
   // Make it a NOP.
   assert_alignment(pc());
-  nop();
+  nop(false);  // 4 bytes
```

Comment:
Normally, the parameters for the assembler functions denotes the operands of the instrunction.
It looks strange to me for the nop() assembler function to take one parameter.
With that consideration, I think it might be better to make this explicit with a incompressible region mark.
Something like:
{
  IncompressibleRegion ir(this);
  nop();
}